src/python/pants/engine/target_test.py

```diff
@@ -772,6 +772,35 @@ class Example(StringSequenceField):
         Example(["hello", 0, "world"], addr)
 
 
+def test_string_sequence_field_valid_choices() -> None:
```

Comment:
This is a copy w/ tweaks (ie use sequence) of `test_string_field_valid_choices()`